src/peergos/server/tests/MultiUserTests.java

```diff
@@ -488,10 +490,43 @@ public void followPeergos() throws Exception {
         UserContext u1 = PeergosNetworkUtils.ensureSignedUp("peergos", "testpassword", network, crypto);
         UserContext u2 = PeergosNetworkUtils.ensureSignedUp("w", "w", network, crypto);
 
+        u2.sendFollowRequest(u1.username, SymmetricKey.random()).get();
+
         List<FollowRequestWithCipherText> u1Requests = u1.processFollowRequests().get();
         assertTrue("Receive a follow request", u1Requests.size() > 0);
     }
 
+    @Test
+    public void sendFeedbackToPeergos() throws Exception {
+        UserContext u1 = PeergosNetworkUtils.ensureSignedUp("peergos", "testpassword", network, crypto);
```

Comment:
peergos is probably a clearer name than u1?

Reply:
Sure is. How about I rename `u1` to `peergos` and `u2` to `newUser`?